src/platform-implementation-js/dom-driver/gmail/gmail-driver/track-gmail-styles.ts

```diff
@@ -5,25 +5,15 @@ import GmailElementGetter from '../gmail-element-getter';
 
 const RGB_REGEX = /^rgb\s*\(\s*(\d+),\s*(\d+),\s*(\d+)\s*\)/;
 
-function getDensity(): string {
-  // get the padding amount on the left-nav-menu entries to find the active
-  // density setting value.
-  const navItemOuter = document.querySelector('.TO .TN');
-  if (!navItemOuter) {
-    Logger.error(new Error('Failed to find nav item outer element'));
-    return 'compact';
-  }
-  const padding = parseInt(
-    getComputedStyle(navItemOuter).getPropertyValue('padding-top'),
-    10
-  );
-  if (padding >= 6) {
-    return 'comfortable';
-  } else if (padding >= 3) {
-    return 'cozy';
-  } else {
-    return 'compact';
```

Comment:
Because we at Streak use these class names, we'd need to keep this existing setup to merge this PR.

That said, @Alex-Chopard, you mentioned

> I was looking for a way to change the style in a custom view when Gmail was in compact mode.

There might be an alternative approach that meets both of our goals. I see a few potential options. In both, we'd keep the newer function.

1. Write new classnames with a different prefix. Instead of `'inboxsdk__gmail_density_' + currentDensity`, we'd have `'inboxsdk__apr2023_gmail_density_' + currentDensity
2. Have the newer function emit to a stream that we expose from the SDK. https://github.com/InboxSDK/InboxSDK/pull/869/files#diff-53e3b61a1eaf27bf59091d62566b209ec90bff97b2622ecb78e203c5392a6ab7R86 shows how we do so with light and dark modes. Maybe

Reply:
An alternative @Macil brought up yesterday was taking our existing internal usages of `/inboxsdk__gmail_density__(cozy|comfortable|compact)/` and flattening down to use `compact` as our default. We'll get back to you on if that change goes through on our end.